test/performance/README.md

```diff
@@ -3,7 +3,7 @@
 Knative performance tests are tests geared towards producing useful performance
 metrics of the knative system. As such they can choose to take a closed-box
 point-of-view of the system and use it just like an end-user might see it. They
-can also go more open-boxy to narrow down the components under test.
+can also go more open-box to narrow down the components under test.
```

Comment:
I don't think the README mentions how you should setup `performance-test-config` secret